core/src/main/java/com/klaytn/caver/kct/kip37/KIP37.java

```diff
@@ -131,11 +128,43 @@ public static KIP37 deploy(Caver caver, String uri, String deployer) throws IOEx
     }
 
     /**
-     * Deploy a KIP-37 contract.
+     * Deploy a KIP-37 contract.<p>
+     * The deployer's keyring should be existed in `caver.wallet`.<p>
+     * If you want to deploy a contract using fee delegation transaction, you can create and send a fee delegated transaction through setting a fee delegation field in SendOptions.
+     * <pre>
+     * <code>
+     *     SendOptions sendOptions = new SendOptions();
+     *     sendOptions.setFrom("deployer address");
+     *     sendOptions.setGas(BigInteger.valueOf(gas value));
+     *     sendOptions.setFeeDelegation(true);
+     *     sendOptions.setFeePayer("fee payer address");
+     *
+     *     KIP37 kip37 = caver.kct.kip37.deploy(uri, sendOptions);
+     * </code>
+     * </pre>
+     * @param caver A Caver instance.
+     * @param uri The URI for token type.
+     * @param sendOptions The send options to deploy a contract.
+     * @return KIP37
+     * @throws IOException
+     * @throws NoSuchMethodException
+     * @throws InstantiationException
+     * @throws ClassNotFoundException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     * @throws TransactionException
+     */
+    public static KIP37 deploy(Caver caver, String uri, SendOptions sendOptions) throws IOException, NoSuchMethodException, InstantiationException, ClassNotFoundException, IllegalAccessException, InvocationTargetException, TransactionException {
```

Comment:
Is there any function that takes `KIP37DeployParams` as a parameter and deploys it?

Based on CommonArchitecture of the Caver, the deploy function of KCT(`caver.kct.kip7` or `caver.kct.kip17`) is to receive `KIPxxDeployParams` and deployer address as parameters.
Can KIP37 also have a function that can receive and deploy `KIP37DeployParams`?

Reply:
The KIP-37 requires only one argument to deploy, I thought it need not to create a class KIP37DeployParams. 
Do you think the KIP37DeployParams needed?

Reply:
This is to make the functional interface the same.
To deploy KCT, the user can think they need to pass the instance(`KIPxxDeployParams`) where the parameters can be defined and the address of the deployer (or `sendOptions`) as parameters according to the function interface of KCT deploy defined in common architecture of Caver.

Reply:
okay, I'll define it.